tools/src/main/java/io/github/ygojson/tools/dataprovider/impl/yugipedia/model/api/params/PipeSeparated.java

```diff
@@ -2,13 +2,9 @@
 
 import java.util.List;
 
-import lombok.AccessLevel;
-import lombok.RequiredArgsConstructor;
-
 /**
  * Helper method to join properly multiple parameters with pipes.
  */
-@RequiredArgsConstructor(access = AccessLevel.PRIVATE)
```

Comment:
In this class, create the private constructor and change the current one for a create method